templates2/display/silo.html

```diff
@@ -208,16 +212,33 @@ <h4 class="modal-title" id="popuptitle"></h4>
         <div class="panel-heading">
             <h3 class="panel-title"><a href="/silo_edit/{{ silo.id }}">{{silo.name}}</a></h3>
             <small>{{ silo.description|default:"" }}</small>
+            {% if tasks_running %}
+                <h4>Background processes runnning</h4>
+                <small style="color:#ff3019">
+                    Data imports are still running in the background.
+                    Silo is unavailable until all processes are finished. # of processes: {{ tasks_running }}
+                    <br> Refresh this page to check the current status.
+                </small>
+            {% endif %}
+            {% if tasks_failed %}
+                <h4 style="color:#ff3019">Background processes failed</h4>
+                <small style="color:#ff3019">
+                    {{ tasks_failed }} background import(s) failed. Data in this silo might be incomplete!
```

Comment:
This message sounds more debugging than informing the user. Try to write something simpler for users.

Reply:
Yes, it is very technical. But the csv import is more focussed on expert users, so i think a more technically direct message is ok.

Reply:
I don't think the user needs to know that a background task/process failed and I don't think they know what silo is. You could just say `The import process failed because the table data might be incomplete.`